lightningd/options.c

```diff
@@ -533,6 +533,9 @@ void register_opts(struct lightningd *ld)
 			 "RRGGBB hex color for node");
 	opt_register_arg("--alias", opt_set_alias, NULL, ld,
 			 "Up to 32-byte alias for node");
+	opt_register_noarg("--test-daemons-only",
+			   opt_set_bool, &ld->test_daemons_only,
+			   "Only test daemons, then exit");
```

Comment:
You could call test_daemons directly from the arg parsing, and make it an early_arg; especially since handle_opts() will create ~/.lightning which I don't think we want here.

Also, you can use the "magic" description opt_hidden, so this isn't displayed with --usage.  I think that's appropriate for these kind of options.

Reply:
okay, although I will have to make `test_daemons` non-`static`.